push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/APNsPushNotificationSender.java

```diff
@@ -208,10 +208,10 @@ private Date createFutureDateBasedOnTTL(int ttl) {
 
         // no TTL was specified on the payload, we use the MAX Default from the APNs library:
         if (ttl == -1) {
-            return new Date(System.currentTimeMillis() + EnhancedApnsNotification.MAXIMUM_EXPIRY * 1000L);
+            return new Date(EnhancedApnsNotification.MAXIMUM_EXPIRY * 1000L);
```

Comment:
Hmm, I'd prefer to use the project specific reference as @sebastienblanc did, since that can change in the future.